### PR TITLE
Support multiple orgs on home screen

### PR DESCRIPTION
**Pull request by jonahkagan**
Also did some redesign of the home and login screens while I was at it. I mostly just reorganized the code from `CreateAudit` into a few different components.

I also found a bug where the audit list doesn't reload after you create a new audit, so I fixed that as well.

Screen Shot 2020-08-31 at 2 47 58 PM

image

Screen Shot 2020-09-03 at 11 53 01 AM

### CODE REVIEW

**Comment by jonahkagan** [comment]
@MorganLove would love to know your initial thoughts before I start in on the tests

**Comment by MorganLove** [comment]
Screenshot from 2020-09-01 08-16-29

@jonahkagan I love the look on the login screen and the audit admin screen! I think the jurisdiction admin screen got a bit messed up in the process though.

File: client/src/components/HomeScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+          State-level audit administrators:{' '}
+          <a href="/auth/auditadmin/start">Log in as an admin</a>
+        </p>
+      </div>
+    </LoginWrapper>
+  )
+}
+
+const ListAuditsWrapper = styled.div`
+  padding: 30px;
```
Comment (MorganLove) [inline client/src/components/HomeScreen.tsx:94]:
```suggestion
  position: relative;
  width: 100%;
  padding: 30px;
```

File: client/src/components/HomeScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+`
+
```
Comment (MorganLove) [inline client/src/components/HomeScreen.tsx:96]:
```suggestion

const JurisdictionAuditsWrapper = styled.div`
  display: inline-block;
  position: absolute;
  margin: 50px;
  left: 50%;
  transform: translate(-50%, -50%);
  text-align: center;
`

```

File: client/src/components/HomeScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+              ))
+            )}
+          </div>
+        ))}
+      {meta.type === 'jurisdiction_admin' &&
+        meta.jurisdictions.map(({ id, name, election }) => (
+          <AuditLink key={id} to={`/election/${election.id}`} intent="primary">
+            {name} - {election.auditName}
+          </AuditLink>
+        ))}
```
Comment (MorganLove) [inline client/src/components/HomeScreen.tsx:137]:
```suggestion
      {meta.type === 'jurisdiction_admin' && (
        <JurisdictionAuditsWrapper>
          <h2>Audits</h2>
          {meta.jurisdictions.map(({ id, name, election }) => (
            <AuditLink
              key={id}
              to={`/election/${election.id}`}
              intent="primary"
            >
              {name} - {election.auditName}
            </AuditLink>
          ))}
        </JurisdictionAuditsWrapper>
      )}
```

**Comment by jonahkagan** [comment]
Fixed the jurisdiction view and added a screenshot. It got a bit complicated because a JA can be the admin for multiple jurisdictions within an audit or across multiple audits.

**Comment by jonahkagan** [comment]
@MorganLove I fixed up the JA screen. Decided not to center it so that it's consistent with the rest of the app (plus a designer once told me I should never center anything except a login screen, and it's been pretty good advice)

Also found a bug where the audit list doesn't refresh after you create a new audit. Came up with a quick slightly hacky solution, which I commented in the code. Let me know your thoughts. I didn't want to try to do a larger-scale solution for now

**Comment by jonahkagan** [comment]
Also added some initial tests

**Comment by MorganLove** [comment]
Looking great!

**Comment by jonahkagan** [comment]
@MorganLove ready for review!

**Review by MorganLove — APPROVED** [review]

**Pull request merged**